app-saml/config/authsources.php

```diff
@@ -0,0 +1,396 @@
+<?php
```

Comment:
I've removed the unnecessary comments down below and also may re-jigg the two templates a bit to merge them into one, since the only difference (at the moment) is on the auth plugin. Will add in the configuration option for the plugins as well and ping you when I finalise that. Not sure if I can move this entirely into the pillars but will see what I can do.